apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -97,6 +128,17 @@ class ProgressLessonTeacherInfo extends React.Component {
             onChange={this.onClickHiddenToggle}
           />
         )}
+        {showGoogleClassroomButton && (
+          <div style={{...styles.buttonContainer, marginBottom: 5}}>
```

Comment:
NIT: rather than adding this style inline, consider adding another style to the styles object with just marginBottom: 5 and using spread to combine the two.

Reply:
👍